src/hotspot/share/jvmci/jvmciCompilerToVM.cpp

```diff
@@ -451,7 +451,7 @@ C2V_VMENTRY_NULL(jobject, getResolvedJavaType0, (JNIEnv* env, jobject, jobject b
 
   const char* base_desc = nullptr;
   JVMCIKlassHandle klass(THREAD);
-  if (offset == oopDesc::klass_offset_in_bytes()) {
+  if (offset == 1 /*oopDesc::klass_offset_in_bytes()*/) {
```

Comment:
@rkennke I'm seeing a crash internally in Graal and I'm suspecting that this line may be the culprit. The error is:

```
Caused by: java.lang.IllegalArgumentException: Unexpected arguments: Ljdk/vm/ci/hotspot/DirectHotSpotObjectConstantImpl; 8 true
	at jdk.internal.vm.ci/jdk.vm.ci.hotspot.CompilerToVM.getResolvedJavaType0(Native Method)
	at jdk.internal.vm.ci/jdk.vm.ci.hotspot.CompilerToVM.getResolvedJavaType(CompilerToVM.java:1020)
...
```
I'm wondering if the commented part of the condition was supposed to be uncommented at some point.